service/ui.go

```diff
@@ -180,13 +180,14 @@ func (uis *UIServer) GetCommonViewData(w http.ResponseWriter, r *http.Request, n
 	if needsProject {
 		var project *model.Project
 		project, err = projectCtx.GetProject()
-		if err != nil || project == nil {
-			if err == nil && project == nil {
-				err = errors.New("no project found")
-			}
+		if err != nil {
 			grip.Errorf(errors.Wrap(err, "no project attached to request").Error())
 			return ViewData{}
 		}
+		if project == nil {
+			grip.Error(errors.New("no project found").Error())
```

Comment:
omit the .Error() at the very least. I might also consider logging the request info (project name, path, or request id)